### DISCUSSION TITLE
TF2?

### DISCUSSION THREAD

**Discussion by BowrainDash** [samisalreadytaken/vs_library#17] [General]
Is there any way I can make this compatible with tf2 vscripting?

**Comment by samisalreadytaken** [comment]
What in particular are you looking for? The math library is already compatible after a simple regex replacement (also available [here](https://raw.githubusercontent.com/samisalreadytaken/keyframes/mapbase/csgo/scripts/vscripts/vs_math.nut)). From others, the only useful utility could be `EventQueue` and `PrintStack`, which aren't hard to copy-paste.

**Reply by BowrainDash** [reply]
Thanks I needed the regex replacement which I couldn't find
